src/Controls/src/Core/Shell/Shell.cs

```diff
@@ -80,10 +97,20 @@ static void OnSearchHandlerPropertyChanged(BindableObject bindable, object oldVa
 				SetInheritedBindingContext(newHandler, bindable.BindingContext);
 		}
 
-		/// <summary>Bindable property for attached property <c>FlyoutItemIsVisible</c>.</summary>
+		/// <summary>
+		/// The <see cref = "FlyoutItem" /> visibility.
+		/// Flyout items are visible in the flyout by default.
+		/// </summary>
 		public static readonly BindableProperty FlyoutItemIsVisibleProperty =
 			BindableProperty.CreateAttached("FlyoutItemIsVisible", typeof(bool), typeof(Shell), true, propertyChanged: OnFlyoutItemIsVisibleChanged);
 		public static bool GetFlyoutItemIsVisible(BindableObject obj) => (bool)obj.GetValue(FlyoutItemIsVisibleProperty);
+
+		/// <summary>
+		/// Sets the visibility of <see cref = "FlyoutItems" />.
+		/// Flyout items are visible in the flyout by default. However, an item can be hidden in the flyout with the FlyoutItemIsVisible property.
+		/// </summary>
+		/// <param name="obj">The object that sets the visibility of flyout items.</param>
+		/// <param name="isVisible">true to set the flyout item as visible; otherwise, false.</param>
```

Comment:
nit: use `<see langword=.../>` for booleans

Reply:
Added everywhere using boolean parameters.